packages/tauri/src-tauri/tauri.conf.json

```diff
@@ -57,7 +57,10 @@
             ]
         },
         "updater": {
-            "active": false
+            "active": true,
+            "endpoints": ["https://github.com/padloc/padloc/releases/download/tauri-latest/tauri-update.json"],
```

Comment:
This URL won't really exist until we publish this tag. We'll also need to manually tag something as latest (which we can do in a follow-up "release" action, which can be the existing one, renamed, which would delete the current latest and tag the new code as such). We might want to change this to `latest` instead of `tauri-latest` if we don't want to consider those separate releases.

Reply:
Does this mean we have to publish a separate tag named `tauri-latest` (or simply `latest`), in addition to the regular releases? What is the advantage over simply using `https://github.com/padloc/padloc/releases/latest/download/tauri-update.json`, which automatically resolves to the latest release?

Reply:
If we keep it as is, yes. Since this is not really part of "latest" right now, I kept it separate, but if we want to move forward with this (and keep Tauri as a target like Electron, for the same version), then we don't need this tag separation at all.

Reply:
I definitely want to have everything in a single release, as discussed above. Previously, the separate Tauri release was really just a way to make it very clear that the Tauri app was still super experimental. Going forward it should be released alongside everything else and eventually replace the Electron version altogether.

Reply:
Ok. Since we already have a v3 release going and I don't want to create confusion, we'll continue with the `tauri-` prefix here for now. We'll remove the prefix once this script becomes a wider release, if that makes sense.